assembly/src/main/scala/org/clulab/reach/assembly/export/AssemblyExporter.scala

```diff
@@ -423,6 +468,30 @@ object AssemblyExporter {
   val CONTEXT_CELLULAR_COMPONENT = "CONTEXT (CELLULAR COMPONENT)"
   val CONTEXT_TISSUE_TYPE = "CONTEXT (TISSUE TYPE)"
   val TRIGGERS = "TRIGGERS"
+  val TRANSLOCATION_SOURCE = "TRANSLOCATION (SOURCE)"
+  val TRANSLOCATION_DESTINATION = "TRANSLOCATION (DESTINATION)"
+
+  // columns for the CMU tabular format
```

Comment:
Maybe these should be under export/src/main/scala/org/clulab/reach/export/cmu/CMUExporter.scala?